### DISCUSSION TITLE
Is there any schedule for Azure Repositories support?

### DISCUSSION THREAD

**Discussion by pharauj** [vercel/vercel#6425] [Help] [answered]
I'm using Azure Repositories to store my projects code. Is there any schedule plan to add login with Azure?

**Comment by leo** [comment, marked as answer]
Hey! Thanks for your feedback!

There are no plans for adding support for it at this point in time, but most definitely in the future.

However, you could already use [Vercel CLI](https://vercel.com/cli) and a custom CI like [Azure Pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/) to deploy from Azure Repositories.
